XCUITests/FxScreenGraph.swift

```diff
@@ -501,42 +529,42 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.tap(table.cells["ChangePasscode"], to: ChangePasscodeSettings, if: "passcode != nil")
     }
 
-    func typePasscode(_ passCode: String) {
-        passCode.forEach { char in
+    func type(_ text: String) {
```

Comment:
I had suggested: 
```swift
func type(text: String) {}
```

in line with Swift 4's naming conventions. This would mean callers type

```swift
type(text: "")
```

instead of 


```swift
type("")
```

Reply:
Oh I see, okay, sorry for the misunderstanding